about.md

```diff
@@ -6,138 +6,80 @@ title: About MDAnalysis
-require a working C compiler. The minimum required version of Python is
-2.7. MDAnalysis has been successfully used on Linux and Mac OS X.
 
-The **MDAnalysis 'Atom' Logo** was designed by **Christian Beckstein** and is
-licensed under a [Creative Commons Attribution-NoDerivs 3.0 Unported
-License](http://creativecommons.org/licenses/by-nd/3.0/).
+## Community
+
+**MDAnalysis** is developed and maintained as a freely available, open-source
+project by a global community of scientists. The MDAnalysis community adheres
+to our [Code of Conduct]({{site.baseurl}}/pages/conduct/) and invites everyone
+to [participate]({{site.baseurl}}/#participating) --- be it on the mailing
+lists, through issue reports, or code contributions.
+
+All *contributors* to the MDAnalysis library and its subprojects are acknowledged
+in a file called `AUTHORS` in each source code repository and in the list of
+contributions; as examples see the [`AUTHORS` file for
+mdanalysis](https://github.com/MDAnalysis/mdanalysis/blob/develop/package/AUTHORS)
+and the [contributors for
+MDAnalysis/mdanalysis](https://github.com/MDAnalysis/mdanalysis/graphs/contributors).
+
+MDAnalysis and the algorithm implemented in the library and the subprojects are
+scientifc software that are described in [academic
+publications]({{site.baseurl}}/pages/citations/).
+
+
+## Open source
+
+All our [code]({{site.baseurl}}/#availability) and our [teaching
+materials]({[site.baseurl}}/pages/learning_MDAnalysis/) are available under
+open source licenses from repositories at
+[https://github.com/MDAnalysis/](https://github.com/MDAnalysis/) and through
+the popular ``pip`` and ``conda`` package managers as well as some Linux
+distributions.
+
+
+## Governance
+
+Project leadership is currently provided by a subset of contributors, the *Core
+Developers*
+([@MDAnalysis/coredevs](https://github.com/orgs/MDAnalysis/teams/coredevs)) who
+have produced substantial contributions over extended lengths of time and who
+remain active in reviewing issues and discussions on the various mailing
+lists. Core Developers are granted commit rights (write access) to the GitHub
+source code repository. New Core Developers are elected with a simple majority
+of current Core Developers.
+
+*In general, all project decisions are made through consensus among the Core
+Developers.*
+
+A rotating subset of three Core Developers is tasked to respond to and to
+investigate Code of Conduct violations.
+
+
+## Partners
+
+MDAnalysis is a [*NumFOCUS-affiliated
+project*](https://www.numfocus.org/affliated-projects.html). Through
+[NumFOCUS](https://www.numfocus.org) (and previously the [Python Software
+Foundation](https://www.python.org/psf/)) we have been participating in [Google
+Summer of Code](https://summerofcode.withgoogle.com/) since 2016. 
+
+We are grateful for financial support from the [National Science
+Foundation](https://www.nsf.gov/) and [NumFOCUS](https://www.numfocus.org).
```

Comment:
I'll ask if the formulation with NumFOCUS is OK. I know this is standard in research to cite agencies providing funds but NumFOCUS financial sponsorship normally is a continued sponsorship instead of a one time grant.

Reply:
But we are not fiscally sponsored; instead we had one NumFOCUS small grant (so far).

Reply:
Yes, please ask.

Reply:
All good